master/docs/manual/installation/nine-upgrade.rst

```diff
@@ -127,6 +127,9 @@ A few notes on changes to the configuration of these reporters:
 
 * :bb:reporter:`GitHubStatus` now accepts a ``context`` parameter to be passed to the GitHub Status API.
 
+* :py:class:`~buildbot.status.builder.Results` and the constants :py:class:`~buildbot.status.results.SUCCESS` should be imported as ____#FIXME!!!_____ instead.
+
```

Comment:
If the goal is to remove use of "buildbot.status", what is the replacement for these? These seem common enough that maybe they should just be split to a special file?